src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -526,8 +517,11 @@ private boolean hasReposInFilter() {
         return allReposInFilterExpr.isEmpty();
     }
 
-    private void openPageOfCurrentlySelectedIssue() {
-        TurboIssue issue = getSelectedElement().get().getIssue();
-        ui.getBrowserComponent().showIssue(issue.getRepoId(), issue.getId(), issue.isPullRequest(), false);
+    private TurboIssue getCurrentlySelectedIssue() {
+        return getSelectedElement().get().getIssue();
+    }
+
+    private void openIssuePageMainTab(TurboIssue issue) {
+        ui.getBrowserComponent().showIssueMainTab(issue.getRepoId(), issue.getId(), issue.isPullRequest(), false);
```

Comment:
If not necessary, no. But in this case, if we are opening a tab (instead of switching tab), it is equivalent to force refresh, looking at the code (since url is not equal).

If we are already in the PR page, we don't open the tab (there are guard clauses before the most of the calls). But if you can make it work without force refresh, it would be better.